### ISSUE TITLE
make_results.py is missing

### ISSUE THREAD

**Issue by maryami66** [captanlevi/Contour-Detection-Pytorch#12]
Hi again,
I could train successfully, but for evaluation step, you mentioned that there is make_results.py which should be run. But, there is no such python file in your repository. Is there any?

Thank you!

**Comment by captanlevi** [comment]
Yeah sorry, I see that the file is missing. Gimme a few days to write the code and upload it.

**Comment by maryami66** [comment]
> Yeah sorry, I see that the file is missing. Gimme a few days to write the code and upload it.

Thanks a lot!
However, I can I also use ContourDetectror.py to show the results:

Maybe this is handy:

first I have created `get_val_batch()`  function for dataloader

then following visualize the results

`a = data_object.get_val_batch(10)
    model_name = "model30.pth"
    model = get_model(model_name)
    model = model.to(device)

    bb = model(a[0:1].to(device))

    bb = bb.detach().cpu().numpy()[0]

    bb = bb[0]

    a = a[0].numpy()
    a = np.rollaxis(a, 2)
    a = np.rollaxis(a, 2)
    plt.imshow(a)
    plt.show()

    bb[bb > .5] = 1
    bb[bb <= .5] = 0

    plt.imshow(bb)
    plt.show()`
